engine/build.gradle

```diff
@@ -15,20 +15,23 @@ buildscript {
         jcenter()
         mavenCentral()
 
-        // HACK: Needed for NUI reflections
+        // HACK: Needed for NUI and gestalt entity-component reflections
         mavenLocal()
         google()
         maven { url "http://artifactory.terasology.org/artifactory/virtual-repo-live" }
+        // Needed for Jsemver, which is a gestalt dependency
+        maven { url = 'https://heisluft.de/maven/' }
```

Comment:
While I'm happy to keep this here, would it be more appropriate to add it to gestalt itself? Or is it needed here specifically?

Reply:
This adds the Jsemver repository to the build script so that the JSemver library can be retrieved. This only applies to the build script itself, rather than the built engine module. Also see the existing comment above this change, which would imply that dependency should be declared in the same file that it is used in.
https://github.com/MovingBlocks/DestinationSol/blob/120014b62889594d00dbacc5af3bcf4aba9edd3c/engine/build.gradle#L11-L13
Without this line, the build script can't resolve the JSemver dependency, causing the build to fail.
```
> A problem occurred configuring project ':engine'.
  > Could not resolve all artifacts for configuration ':engine:classpath'.
  > Could not find com.github.zafarkhaja:java-semver:0.10.0.
```
The repositories used to resolve runtime dependencies are stored in `config/gradle/common.gradle` instead. I don't think that gestalt can define which repositories to obtain its dependencies from using published library metadata but I don't know enough about the systems used to be sure.